.github/workflows/ci.yaml

```diff
@@ -168,7 +168,7 @@ jobs:
           cat docker-compose.yml
           echo "run kwild in docker"
           docker compose up -d
-          chain_id=`curl -s http://localhost:8080/api/v1/chain_info | jq .chain_id`
+          chain_id=`curl -s http://localhost:8080/api/v1/chain_info | jq -r .chain_id`
```

Comment:
Maybe sleep for 3 seconds, kwil-db could be ramping up and not ready

Reply:
Clone and build kwil-js while you wait :)